ShareX.UploadersLib/Forms/UploadersConfigForm.cs

```diff
@@ -558,6 +559,21 @@ public void LoadSettings()
             txtPomfUploadURL.Text = Config.PomfUploader.UploadURL;
             txtPomfResultURL.Text = Config.PomfUploader.ResultURL;
 
+            // Seafile
+
+            txtSeafileAPIURL.Text = Config.SeafileAPIURL;
+            txtSeafileAuthToken.Text = Config.SeafileAuthToken;
+            txtSeafileDirectoryPath.Text = Config.SeafilePath;
+            txtSeafileLibraryPassword.Text = Config.SeafileEncryptedLibraryPassword;
+            txtSeafileLibraryPassword.ReadOnly = (Config.SeafileIsLibraryEncrypted ? true : false);
+            btnSeafileLibraryPasswordValidate.Enabled = (Config.SeafileIsLibraryEncrypted ? false : true);
```

Comment:
That sub item is for encrypted folders.  Try adding an encrypted folder to
your account and you will see what I'm talking about.
On Oct 13, 2015 2:33 AM, "Jaex" <notifications@github.com> wrote:

> In ShareX.UploadersLib/Forms/UploadersConfigForm.cs
> <https://github.com/ShareX/ShareX/pull/1061#discussion_r41830958>:
>
> > @@ -558,6 +559,21 @@ public void LoadSettings()
> >              txtPomfUploadURL.Text = Config.PomfUploader.UploadURL;
> >              txtPomfResultURL.Text = Config.PomfUploader.ResultURL;
> >
> > +            // Seafile
> > +
> > +            txtSeafileAPIURL.Text = Config.SeafileAPIURL;
> > +            txtSeafileAuthToken.Text = Config.SeafileAuthToken;
> > +            txtSeafileDirectoryPath.Text = Config.SeafilePath;
> > +            txtSeafileLibraryPassword.Text = Config.SeafileEncryptedLibraryPassword;
> > +            txtSeafileLibraryPassword.ReadOnly = (Config.SeafileIsLibraryEncrypted ? true : false);
> > +            btnSeafileLibraryPasswordValidate.Enabled = (Config.SeafileIsLibraryEncrypted ? false : true);
>
> ReadOnly one not make sense here and for validate one you could have just
> used !Config.SeafileIsLibraryEncrypted
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShareX/ShareX/pull/1061/files#r41830958>.
>